### DISCUSSION TITLE
Improvements on Medic AI (No Morphine Option)

### DISCUSSION THREAD

**Discussion by Holgerhollebolle** [acemod/ACE3#9080] [Ideas]
I am playing with KAT Medic system. Unfortunatly I can not use Medic AI as they tend to use Morphine to much. This makes AI pass out because of low BP.  I would like a setting for disabling AI using Morphine.
Also I would vote for AI consuming Medical Items as currently they dont which makes Medic Logistics Unnecesarry.

**Comment by johnb432** [comment]
#3594
